Holub (#279)
aspose-pdf/Aspose.PDF-Documentation: net/advanced-operations/working-with-documents/working-with-layers/_index.md

    }
```

> Since the 24.9 release, this feature has been updated.

Reviewer: ```suggestion
The 24.9 release has resulted in an update to this feature.
```
